### ISSUE TITLE
Version using iron:router and yield

### ISSUE THREAD

**Issue by rdewolff** [chip/html5up-strata#1]
Have you tried making a version using iron:router and templates split in different files?

**Comment by chip** [comment]
No, but would be happy to review a Pull Request from you or anyone who wants to take a stab at it.
